### ISSUE TITLE
WTHIT does not show any info for Productive Bees or Resource Chickens

### ISSUE THREAD

**Issue by Sirfirelord1** [badasintended/wthit#257]
### Describe the issue

There should be tooltip info for them, like breeding cooldown, time to grow up, etc. But for some reason WTHIT does not show these (Jade can show them though)

Steps to reproduce: Install the above mods + WTHIT + badpackets
look at a modded bee or chicken

### Log output and crash report

NA

### Additional context

https://github.com/Sirfirelord1/Petty-s-Cozycraft/issues/10

**Comment by deirn** [comment]
Pretty sure WTHIT has breeding cooldown and grow timer. What exactly that you want?

See [`MobTimer`](https://github.com/badasintended/wthit/blob/dev/master/src/pluginVanilla/java/mcp/mobius/waila/plugin/vanilla/provider/MobTimerProvider.java).

**Comment by Sirfirelord1** [comment]
Definitely not. I've explored all in-game configs, as well as all config files. No option to enable, and the tooltip still does not show

**Comment by deirn** [comment]
Right, I guess both the mods are not using vanilla method to do the age for some reason. I'll redirect this issue to [Megane](https://github.com/badasintended/megane) as there's where the mod compat goes.
